packages/fluentui/react-northstar/src/components/Accordion/Accordion.tsx

```diff
@@ -102,8 +102,6 @@ class Accordion extends AutoControlledComponent<WithAsProp<AccordionProps>, Acco
 
   static deprecated_className = accordionClassName;
```

Comment:
You said in the description "Replace all component's className (deprecated_className)", but I see them still in place, is that OK?

Reply:
Or are we keeping `deprecated_className` for class components?

Reply:
We still need them internally for the components that extending the `UIComponent` and `AucontrolledComponents`, but we don't want users to rely on those.